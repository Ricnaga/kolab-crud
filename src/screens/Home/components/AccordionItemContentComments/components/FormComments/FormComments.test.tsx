import {
  beforeEach,
  describe,
  expect,
  it,
  renderWithTheme,
  vi,
} from "@vitest-utils";
import { FormComments } from ".";

describe("Home Component: AccordionItemContentComments", () => {
  beforeEach(() => vi.resetModules());

  it("should render correctly", () => {
    const { container } = renderWithTheme(
      <FormComments
        comment={{
          body: "BODY_MOCKED",
          email: "EMAIL_MOCKED",
          id: 1,
          name: "NAME_MOCKED",
          postId: 1,
        }}
        postId="1"
      />
    );

    expect(container).toMatchSnapshot();
  });
});
