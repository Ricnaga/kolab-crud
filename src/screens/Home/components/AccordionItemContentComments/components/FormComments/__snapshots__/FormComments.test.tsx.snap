// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Home Component: AccordionItemContentComments > should render correctly 1`] = `
<div>
  <div
    class="css-1f9j1y1"
  >
    <form>
      <div
        class="css-1jq3den"
      >
        <div
          class="chakra-stack css-8g8ihq"
        >
          Usuário:
           
          <em
            class="css-b6ieky"
          >
            NAME_MOCKED
          </em>
          Email:
           
          <em
            class="css-b6ieky"
          >
            EMAIL_MOCKED
          </em>
          Descrição:
           
          <em
            class="css-b6ieky"
          >
            BODY_MOCKED
          </em>
        </div>
        <div
          class="css-t9y6jz"
        >
          <button
            class="chakra-button css-uom3v7"
            type="submit"
          >
            <svg
              fill="currentColor"
              height="1em"
              stroke="currentColor"
              stroke-width="0"
              viewBox="0 0 24 24"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M0 0h24v24H0z"
                fill="none"
              />
              <path
                d="M3 17.25V21h3.75L17.81 9.94l-3.75-3.75L3 17.25zM20.71 7.04a.996.996 0 0 0 0-1.41l-2.34-2.34a.996.996 0 0 0-1.41 0l-1.83 1.83 3.75 3.75 1.83-1.83z"
              />
            </svg>
             
            Editar
          </button>
          <button
            class="chakra-button css-1ogt38g"
            type="button"
          >
            <svg
              fill="currentColor"
              height="1em"
              stroke="currentColor"
              stroke-width="0"
              viewBox="0 0 24 24"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M0 0h24v24H0z"
                fill="none"
              />
              <path
                d="M6 19c0 1.1.9 2 2 2h8c1.1 0 2-.9 2-2V7H6v12zM19 4h-3.5l-1-1h-5l-1 1H5v2h14V4z"
              />
            </svg>
             Remover
          </button>
        </div>
      </div>
    </form>
  </div>
</div>
`;
